packages/turf-invariant/test.js

```diff
@@ -300,8 +300,12 @@ test("invariant -- getCoords", (t) => {
 
   t.throws(() => invariant.getCoords(false));
   t.throws(() => invariant.getCoords(null));
-  t.throws(() => containsNumber(invariant.getCoords(["A", "B", "C"])));
-  t.throws(() => containsNumber(invariant.getCoords([1, "foo", "bar"])));
+  t.throws(() =>
```

Comment:
These were actual bugs in the tests, it would throw an error before because containsNumber was undefined